### PR TITLE
IAN: Add HttpCache::repair()

### PR DESCRIPTION
**Pull request by mlogan**
It's very hard to test all the different error cases I tried to handle here - please scrutinize my error handlers for typos and missing variables.

### CODE REVIEW

**Comment by BKcore** [comment]
LGTM

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "main": "index.js",
   "scripts": {
-    "test": "nodeunit test.js"
+    "test": "./node_modules/.bin/nodeunit test.js"
```
Comment (statico) [inline package.json:10]: You shouldn't need this with `npm run test`. Using `npm run` is supposed to add all node_modules executables to your path automatically.

**Comment by statico** [comment]
GLHF but questions in person

Consider adding docs on this method to the README

**Comment by BKcore** [comment]
GLHF

File: package.json (hunk 1 above)
Comment (mlogan) [inline package.json:10]: But I had a global nodeunit. Doing it this way seems better in some cases and worse in none.

Comment (statico) [inline package.json:10]: `npm run` puts the `.bin` early in the path, so if you did an `npm install` in this repo dir, `npm test` should use the version of `nodemon` that's specified in `devDepenencides`.

**Pull request merged**